assets/prototype/domain/eventEditor/data/types.ts

```diff
@@ -17,15 +17,18 @@ export interface Datetime extends Entity {
 	description?: string;
 	endDate?: string;
 	isActive?: boolean;
+	isDeleted?: boolean; // TODO: refactor according to resolution from here: https://github.com/eventespresso/event-espresso-core/issues/2089
 	isExpired?: boolean;
 	isPrimary?: boolean;
 	isSoldOut?: boolean;
 	isUpcoming?: boolean;
 	length?: number;
 	order?: number;
+	regLimit?: number;
 	reserved?: number;
 	sold?: number;
 	startDate?: string;
+	status?: string;
```

Comment:
What does `status` represent?

Reply:
I've ported over from places like these: 
![image](https://user-images.githubusercontent.com/1477453/71416086-ab8f3e80-2667-11ea-8ad3-0c78faa34a78.png)

Reply:
and this:
![image](https://user-images.githubusercontent.com/1477453/71416141-ebeebc80-2667-11ea-9853-eb630f107877.png)

Reply:
if that shouldn't be there I can remove that

Reply:
That was in JS Models and is no more relevant I think.